lib/auth/clt.go

```diff
@@ -407,12 +407,21 @@ func (c *Client) GetCertAuthority(ctx context.Context, id types.CertAuthID, load
 }
 
 // DeleteCertAuthority deletes cert authority by ID
-func (c *Client) DeleteCertAuthority(id types.CertAuthID) error {
+func (c *Client) DeleteCertAuthority(ctx context.Context, id types.CertAuthID) error {
 	if err := id.Check(); err != nil {
 		return trace.Wrap(err)
 	}
-	_, err := c.Delete(context.TODO(), c.Endpoint("authorities", string(id.Type), id.DomainName))
-	return trace.Wrap(err)
+
+	err := c.APIClient.DeleteCertAuthority(ctx, id)
+	switch {
+	case err == nil:
+		return nil
```

Comment:
nit: `trace.Wrap()` checks internally for `nil`, so this case is the same as `default`.